ui/apps/dashboard/src/app/api/inngest/functions/agents/query-writer.ts

```diff
@@ -0,0 +1,96 @@
+import { createAgent, createTool, openai, type Network } from '@inngest/agent-kit';
+import { z } from 'zod';
+
+import type { InsightsAgentState as InsightsState } from './event-matcher';
+import type { GenerateSqlInput, GenerateSqlResult } from './types';
+
+function sanitizeSql(text: string): string {
```

Comment:
What happens if this function fails to properly sanitize the query? Is there a security risk?